pyramid_oereb/contrib/models/interlis_2_3/main.py

```diff
@@ -0,0 +1,702 @@
+    document_id = Column('dokument_dokument',
+                         Integer,
+                         ForeignKey=Document.t_id,
+                         nullable=True)
+    logo_id = Column('logo_bild',
+                     Integer,
+                     ForeignKey=Logo.t_id,
+                     nullable=True)
+    document = relationship(
+            'Document',
+            backref='multilingual_blob'
+    )
+    logo = relationship(
+            'Logo',
+            backref='multilingual_blob'
+    )
+
+
+class LocalisedBlob(Base):
+    """
+    Bucket to resolve entries related to OeREBKRM_V2_0.LocalisedBlob.
+
+    Attributes:
+        t_id (int): The identifier. This is used in the database only and must not be set manually. If
+            you  don't like it - don't care about.
+        language (str): Interlis: InternationalCodes_V1.LanguageCode_ISO639_1.
+        blob (large binary): Interlis: BLACKBOX BINARY
+        multilingualblob_id (int): The foreign key to the multilingual blob this localised blob
+            is related to.
+    """
+    __table_args__ = {'schema': app_schema_name}
+    __tablename__ = 'localisedblob'
+    t_id = Column(Integer, primary_key=True, autoincrement=False)
+    language = Column('alanguage', String, nullable=True)
+    blob = Column('ablob', LargeBinary, nullable=False)
+    multilingualblob_id = Column(
+        'multilingualblob_localisedblob',
+        Integer,
+        ForeignKey(MultilingualBlob.t_id),
+        nullable=True
+    )
+
+
+# Not part of OeREBKRMkvs_V2_0 --> created in import_main_structure.py
+class Municipality(Base):
+    """
+    The municipality is the place where you hold the information about all the municipalities you are having
+    in your canton. This is used also in the applications process to check whether a municipality is published
+    or not.
+
+    Attributes:
+        fosnr (int): The identifier of the municipality. It is the commonly known id_bfs or as or
+            nofs in the  french part.
+        name (str): The Name of the municipality.
+        published (bool): Switch whether a municipality is published or not. This has direct
+            influence on extract  generation.
+        geom (geoalchemy2.types.Geometry): The geometry of municipality borders. For type
+            information see geoalchemy2_.  .. _geoalchemy2:
+            https://geoalchemy-2.readthedocs.io/en/0.2.4/types.html  docs dependent on the
+            configured type.  This concrete one is MULTIPOLYGON
+    """
+    __table_args__ = {'schema': app_schema_name}
+    __tablename__ = 'municipality'
+    fosnr = Column(Integer, primary_key=True, autoincrement=False)
+    name = Column(String, nullable=False)
+    published = Column(Boolean, nullable=False, default=False, server_default=Text('FALSE'))
+    geom = Column(Geometry('MULTIPOLYGON', srid=srid), nullable=True)
+
+
+# Not part of OeREBKRMkvs_V2_0 --> created in import_main_structure.py
+class RealEstate(Base):
```

Comment:
Maybe have a look at my PR to see how to avoid copy of SQLAlchemy classes: https://github.com/openoereb/pyramid_oereb/pull/1317/files#diff-0490ddcd6ab2e9839fae9110dd8077daf40fd03665a8f027518733e8bb8f3255R28